site/public/js/user-profile.js

```diff
@@ -284,6 +284,7 @@ $(document).ready(() => {
 
                     // Check user's current time zone, give a warning message if the user's current time zone differs from systems' time-zone
                     const offset = getCurrentUTCOffset();
+                    // eslint-disable-next-line eqeqeq
                     if (response.data.utc_offset != offset) {
```

Comment:
I was able to get the same result when changing the time zone in the user profile after replacing `response.data.utc_offset != offset` with `response.data.utc_offset !== offset`.